src/modules/facility-loan-transaction/dto/get-loan-transaction-params.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+
+export class GetLoanTransactionParamsDto {
+  @ValidatedStringApiProperty({ description: 'The UKEF identifier for the facility.', length: 10, pattern: /^\d{10}$/ })
+  facilityIdentifier: string;
+
+  @ValidatedStringApiProperty({ description: 'The bundle identifier for the loan transaction.', length: 9 })
```

Comment:
I checked dev and prod. Looks like bundleIdentifier is always 10 digits, maybe I said that is 9 before.